asndocument.cpp

```diff
@@ -27,14 +27,26 @@
 
 #include "asnhighlighter.h"
 #include "asn1acnconstants.h"
+#include "asndocumentprocessor.h"
 
 using namespace Asn1Acn::Internal;
 
+const int PROCESS_DOCUMENT_INTERVAL_MS = 1000;
```

Comment:
add `static` or anonymous namespace (although as long as it address is not taken, all consts are treated like "static", it's good to be sure ;) )